bottomTab/Bottom.js

```diff
@@ -0,0 +1,186 @@
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import COLOS from '../constants/colors'
+import Home from '../screens/Home';
+import Profile from '../screens/Profile';
+import Predict from '../screens/Predict';
+const Tab = createBottomTabNavigator();
+
+const tabs = [
+    {
+        name: 'Home',
+        component: Home,
+        label: 'Home',
+        icon: 'home',
+        inActiveIcon: 'home-outline'
+    },
+    {
+        name: 'Scan',
+        component: Predict,
+        label: 'Scan',
+        icon: 'magnify-scan',
+        inActiveIcon: 'line-scan'
+    },
+    {
+        name: 'Profile',
+        component: Profile,
+        label: 'Profile',
+        icon: 'account',
+        inActiveIcon: 'account-outline'
+    },
+];
+
+const { width } = Dimensions.get('window');
+const MARGIN = 13;
+const TAB_BAR_WIDTH = width - 2 * MARGIN;
+const TAB_WIDTH = (TAB_BAR_WIDTH / tabs.length);
+
+function MyTabBar({ state, descriptors, navigation }) {
+    const [translateX] = useState(new Animated.Value(0));
+    const translateTab = (index) => {
+        Animated.spring(translateX, {
+            toValue: index * TAB_WIDTH,
+            useNativeDriver: true,
+        }).start();
+    }
+
+    useEffect(() => {
+        translateTab(state.index)
+    }, [state.index])
+    return (
+        <View style={styles.tabBarContainer}>
+            <Animated.View style={styles.slidingTabContainer}>
+                <Animated.View style={[styles.slidingTab, { transform: [{ translateX }] }]} />
+            </Animated.View>
+            {state.routes.map((route, index) => {
+                const { options } = descriptors[route.key];
+                const label =
+                    options.tabBarLabel !== undefined
+                        ? options.tabBarLabel
+                        : options.title !== undefined
+                            ? options.title
+                            : route.name;
+
+                const isFocused = state.index === index;
+
+                const onPress = () => {
+                    const event = navigation.emit({
+                        type: 'tabPress',
+                        target: route.key,
+                        canPreventDefault: true,
+                    });
+
+                    if (!isFocused && !event.defaultPrevented) {
+                        navigation.navigate(route.name, route.params);
+                    }
+                };
+
+                const onLongPress = () => {
+                    navigation.emit({
+                        type: 'tabLongPress',
+                        target: route.key,
+                    });
+                };
+                const tabBarIcon = options.tabBarIcon;
+                return (
+                    <TouchableOpacity
+                        key={route.key}
+                        accessibilityRole="button"
+                        accessibilityState={isFocused ? { selected: true } : {}}

```

Comment:
the `accessibilityState` prop is optional, you can use `isFocused && { selected: true }` instead of a ternary operator